### ISSUE TITLE
Scrapyard Error re: MUO sidebar

### ISSUE THREAD

**Issue by Kaos-Ian** [GChristensen/scrapyard#92]
6 Best Sites for Cheat Sheets, Shortcuts, and Quick Reference Cards @
https://www.makeuseof.com/best-sites-for-cheat-sheets-shortcuts-quick-reference/

Error name: Error
Error message: n.external_id is undefined
Origin: Sidebar initialization
Browser version: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0

Stacktrace

**Comment by GChristensen** [comment]
Hi, 

More context is necessary. My best guess that this is an error from sidebar. Since there is no stack trace, a screenshot of the error page would be also helpful.

**Comment by GChristensen** [comment]
As far, I can say for now, this error may happen only if Files shelf is enabled. Unless it is fixed, it may be worth to disable the Files shelf if it is enabled.

**Comment by Kaos-Ian** [comment]
Sorrry, I was sauccessful upon reloading the page. Then I made the download without any difficulties. 
The problem arose when I activated the sidebar menu to select an option and then clicked somewhere else on the page, automatically sliding it out of view.
The second try, I did not activate the menu and "BINGO!", no problem.
I will know better in the future.
Thanks for your time, hopefully it will help you in the future should you run across a similar issue.
